### PR TITLE
Add routing to client side app.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
- Provides a simple example of routing working.

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
@clloyd

File: public/components/other.react.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default class ReactApp extends React.Component {
```
Comment (clloyd) [inline public/components/other.react.js]: This should probably be `class OtherComponent` It'll work without renaming, but for debugging it'll surface this name, so nice to keep them unique.

**Comment by clloyd** [comment]
(Routing looks good to me)

**Comment by LATaylor-guardian** [comment]
Thanks @clloyd

**Pull request merged**